core/src/integTest/scala/water/sparkling/itest/local/H2OContextLocalClusterSuite.scala

```diff
@@ -37,9 +37,20 @@ class H2OContextLocalClusterSuite extends FunSuite
     // For distributed testing we need to pass around jar containing all implementation classes plus test classes
     val conf = defaultSparkConf.setJars(swassembly :: Nil)
     sc = new SparkContext("local-cluster[3,2,2048]", "test-local-cluster", conf)
-    hc = H2OContext.getOrCreate(sc, new H2OConf(sc))
+
+    // start h2o cloud in case of external cluster mode
+    if(testsInExternalMode(sc.getConf)){
+      startCloud(3, sc.getConf)
+    }
+
+    hc = H2OContext.getOrCreate(sc, new H2OConf(sc).setNumOfExternalH2ONodes(3))
```

Comment:
This code is more or less repeated in H2OContextLocalSuite, differs only in a constant; makes sense to write a common function.